lib/tasks/rspec.rake

```diff
@@ -1,8 +1,4 @@
-begin
-  require "rspec/core/rake_task"
-
-  RSpec::Core::RakeTask.new(:spec)
-
-  task default: :spec
-rescue LoadError
+desc "RSpec"
```

Comment:
Rather than make this different to other repos, we can get rid of it altogether if we just [put rspec-rails into the dev+test group](https://github.com/alphagov/email-alert-frontend/blob/master/Gemfile#L24).

Reply:
Thanks. Done!